tasks/section5.yml

```diff
@@ -191,15 +297,19 @@
       - rule_5.2.2
 
 - name: "SCORED | 5.2.2 | PATCH | Ensure SSH Protocol is set to 2"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: '^(#)?Protocol \d'
+      line: 'Protocol 2'
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.2
 
 - name: "SCORED | 5.2.3 | AUDIT | Ensure SSH LogLevel is set to INFO"
-  command: /bin/true
+  command: grep "^LogLevel INFO" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*